stepman/util.go

```diff
@@ -65,17 +65,21 @@ func DownloadStep(collection models.StepCollectionModel, step models.StepModel)
 		for key, value := range downloadLocationMap {
 			switch key {
```

Comment:
download a step **only if needed** (if it's already available in local cache then don't do it!)